simulation_starter.py

```diff
@@ -11,59 +11,107 @@ def get_results(config):
 
 
 
-    os.system("g++ math_pendulum.cpp")
+    os.system("g++ math_pendulum.cpp -O2")
```

Comment:
I suggest reading a little more regarding compiler optimization flags. for example [https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html)